tv.plex.PlexHTPC.yml

```diff
@@ -0,0 +1,109 @@
+app-id: tv.plex.PlexHTPC
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/Plex.sh
+finish-args:
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
so 
```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

Reply:
but I'll add a note that I have seen many problems with Qt and Wayland, and if you build Qt into your package (which I think you do other it wouldn't work here, even more reason to use the smaller freedesktop sdk) then make sure this works when running a GNOME session running Wayland.

Reply:
It turns out that we are missing the wayland Qt plugin from our build, so I've commented out wayland for now.